### ISSUE TITLE
SystemJS example is out dated

### ISSUE THREAD

**Issue by rahmathm1** [valor-software/ngx-bootstrap#2259]
SystemJS example provided in doc is out dated. It's done using **ngx-bootstrap**
**package.json**

`"ngx-bootstrap": "^1.8.0",`

**systemjs.config.js** 

```
map: {
     ...
    'ngx-bootstrap': 'node_modules/ngx-bootstrap'
}

packages: {
    ..
    'ngx-bootstrap': { format: 'cjs', main: 'bundles/ngx-bootstrap.umd.js', defaultExtension: 'js' },
}
```

**app.module.ts**

`import { ModalModule } from 'ngx-bootstrap';`

I'm importing `BsModalService` in my component as 

`import { BsModalService } from 'ngx-bootstrap/modal';`

I'm getting this error, what am I missing here.

```
GET http://localhost:8000/node_modules/ngx-bootstrap/modal.js 404 (Not Found)
(index):21 Error: (SystemJS) XHR error (404 Not Found) loading http://localhost:8000/node_modules/ngx-bootstrap/modal.js
```

**Comment by valorkin** [comment]
@rahmathm1 it is nice reason for small PR

plus: my strong advice to migrate from system.js to angular-cli

**Comment by rahmathm1** [comment]
@valorkin Thanks for the reply. But I don't want to use ng-cli as my starting file is a php file in laravel framework and ng-cli generates it's own html file. Could you please tell me how to fix this issue?

**Comment by IlyaSurmay** [comment]
SystemJS users have to import modules from `ngx-bootstrap`, so this `import { BsModalService } from 'ngx-bootstrap/modal';` won't work.
Issue with missing `BsModalService` in `ngx-bootstrap` was fixed in 1.8.1, please update.
